.github/workflows/example_reusable-integration.yaml

```diff
@@ -138,6 +138,13 @@ jobs:
 #       project: my-project
 #       build-name: my-build
 #       version: v1.0.0
+#   sign-artifacts:
```

Comment:
I'm a bit confused... why is dependabot adding commented out stuff?

Reply:
but I think it should be OK. (also this commented out part is implemented in the future PR